src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -529,6 +534,38 @@ private bool IsPreamble()
             return _checkPreamble;
         }
 
+        private StringBuilder AcquireSharedStringBuilder(int capacity)
+        {
+            // Do not touch the shared builder if it will be removed on release
+            if (capacity > MaxSharedBuilderCapacity)
+                return new StringBuilder(capacity);
+
+            // note that since StreamReader does not support concurrent reads it is not needed to
+            // set this._builder to null to avoid parallel acquisitions.
+            StringBuilder sb = this._builder;
```

Comment:
I didn't pick up on this before, but now that we use a private field with a leading underscore, we should no longer use the `this.` prefix.